enterprise/cmd/cody-gateway/shared/config.go

```diff
@@ -112,6 +114,17 @@ func (c *Config) Load() {
 
 	c.ActorConcurrencyLimit.Percentage = float32(c.GetPercent("CODY_GATEWAY_ACTOR_CONCURRENCY_LIMIT_PERCENTAGE", "50", "The percentage of daily rate limit to be allowed as concurrent requests limit from an actor.")) / 100
 	c.ActorConcurrencyLimit.Interval = c.GetInterval("CODY_GATEWAY_ACTOR_CONCURRENCY_LIMIT_INTERVAL", "10s", "The interval at which to check the concurrent requests limit from an actor.")
+
+	thresholds := c.Get("CODY_GATEWAY_ACTOR_RATE_LIMIT_NOTIFY_THRESHOLDS", "90,95,99", "The comma-separated list of the percentage of the rate limit usage to trigger an notification.")
```

Comment:
How about that? This way we would also get notifications once someone is _actually_ rate limited 
```suggestion
	thresholds := c.Get("CODY_GATEWAY_ACTOR_RATE_LIMIT_NOTIFY_THRESHOLDS", "90,95,100", "The comma-separated list of the percentage of the rate limit usage to trigger an notification.")
```

Reply:
Sure, captain :)

Reply:
Addressed in https://github.com/sourcegraph/sourcegraph/pull/53137/commits/21bdbe4c161093a54633dca4e60e2ad403307825